app/models/project.rb

```diff
@@ -94,4 +94,15 @@ def contribulator_url
   def url
     homepage.presence || github_url
   end
+
+  def community_profile(nickname, token, options)
+    GithubClient.new(nickname, token).community_profile(options)
```

Comment:
Can we reuse the `get_github_data` method on project here?

```
get_github_data('community_profile', nickname, token, 1, options)
```

The months ago option doesn't make much sense for this but we can refactor that later